cypress/e2e/API/api.cy.js

```diff
@@ -0,0 +1,14 @@
+/// <reference types="Cypress" />
+
+//API Test Suit
+describe('API Tests', () => {
+
+    it('Recupera clientes com sucesso', () => {
+        //Realiza uma request para a p�gina e verifica se a resposta � 200
+        cy.request('https://engage-sphere.vercel.app').should(function (response) {
+            console.log(response)
+            const { status } = response
+            expect(status).to.equal(200)
+        })
+    })
+})
```

Comment:
É recomendado deixar uma linha em branco ao final de cada arquivo.

Dessa forma, ferramentas como o GitHub não exibem o ícone de `No newline at end of file`.

Isso é uma boa prática e convenção que deve ser seguida.